api-tests/management/external_test.go

```diff
@@ -95,6 +95,11 @@ func TestAddExternal(t *testing.T) {
 					RunsOnNodeID: nodeID,
 					Scheme:       "http",
 					MetricsPath:  "/metrics",
+					MetricsResolutions: &agents.ListAgentsOKBodyExternalExporterItems0MetricsResolutions{
+						Hr: "0s",
+						Lr: "0s",
+						Mr: "0s",
+					},
```

Comment:
I believe that any other places where we don't test MetricsResolution should support nil value for this field

Reply:
Done. Not every resolution field is nullable as well as whole `MetricsResoultions` field